app/src/main/java/nl/tudelft/b_b_w/view/ContactAdapter.java

```diff
@@ -134,9 +134,7 @@ public View getView(final int position, View convertView, ViewGroup parent) {
             view = inflater.inflate(R.layout.simple_list_item_1, null);
```

Comment:
The getview method is becoming too large, we should considering refactoring it in the future. Next to that, the method is pretty complex; could you place comments in the function itself too?